test/Error/rlexe.xml

```diff
@@ -1,5 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
 <regress-exe>
+  <test>
+    <default>
+      <files>error_cause.js</files>
+      <compile-flags>-args summary -endargs</compile-flags>
```

Comment:
You may have guessed but just an FYI this gives `-args summary -endargs` as extra command line arguments for the test.

Which in turn makes WScript.Arguments[0] = "summary" - so that testRunner.runTests gets called with `verbose: false`

The reason for doing this to set the "verbose" option of a test is so you can run the test yourself with `verbose = true` as the default (to print out all the test cases within the file) vs when it's run here through the test runner it prints only failures + the word "pass" or "fail" at the end.